### ISSUE TITLE
Validation issue for +91 8700983012

### ISSUE THREAD

**Issue by anandAbhi** [googlei18n/libphonenumber#1385]
Country/region affected: IN
Example number(s) affected: +91 870 098 3012
The phone number range(s) to which the issue applies: +918700XXXXXX
The type of the number(s): Mobile
The cost, if applicable ("toll-free", "premium rate", "shared cost"): Unknown

We've called this number. This is a **Reliance Jio** number.

http://libphonenumber.appspot.com/phonenumberparser?number=%2B918700983012

**Comment by penmetsaa** [comment]
@anandAbhi ,

Thank you for taking the time to improve libphonenumber!

I have tried calling this number. I got automated message saying the number do not exists.
We are not able to find valid evidences for these numbers online. 

Checked all popular recharge websites (like paytm.com). No website auto populates that it is Reliance Jio sim.

We need more information to fix this. Please see our [contribution
guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md).

**Comment by anbarasuthresnathan** [comment]
@penmetsaa , I too Face the same problem as Anand Mentioned,

I am trying to Validate the Number Which Starts with +91 700 xxxxxxx which is valid Mobile number for the Country region code "IN" .

But When we try to validate the Number we are not able to save the Contact Number as it is mentioned as incorrect Mobile Number

**Comment by keghani** [comment]
@anandAbhi This issue is blocked on your or other GitHub contributors' input. Please help
us find evidence as requested by @penmetsaa. Otherwise we don't have enough information to
fix the issue and may have to close it as infeasible.

@anbarasuthresnathan This issue is about numbers beginning with 8700, not 700.
Please see our [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md) and file a new issue if there is a problem with validation of 700 numbers.

**Comment by veris-abhinavanand** [comment]
Refer: https://en.wikipedia.org/wiki/Mobile_telephone_numbering_in_India#8xxx_Series

![screenshot from 2016-10-27 15-45-05](https://cloud.githubusercontent.com/assets/16878162/19763656/a414d658-9c5c-11e6-9cce-4446c486e332.png)

**J** stands for Jio
**DL** stands for Delhi

Will this be sufficient?

**Comment by anandAbhi** [comment]
Refer: https://en.wikipedia.org/wiki/Mobile_telephone_numbering_in_India#8xxx_Series

![screenshot from 2016-10-27 15-45-05](https://cloud.githubusercontent.com/assets/16878162/19763656/a414d658-9c5c-11e6-9cce-4446c486e332.png)

**J** stands for Jio
**DL** stands for Delhi

Will this be sufficient?

**Comment by lararennie** [comment]
Thanks, helpful. Any idea why we get an automated message but you can ring the number?

**Comment by ankitpopli1891** [comment]
The number belongs to our friend. He rarely uses this number. We made sure the number was active when we called. We can arrange the same for you guys to validate.

**Comment by penmetsaa** [comment]
Thanks for asking :) 
No need actually. We found official evidence which mentions this prefix. 
This fix will be part of upcoming release.

**Comment by penmetsaa** [comment]
This is fixed in release 7.7.4
http://libphonenumber.appspot.com/phonenumberparser?number=%2B918700983012
